gene/gene.go

```diff
@@ -156,17 +178,17 @@ func CdnaPosToCodon(g *Gene, cdnaPos int) (dna.Codon, error) {
 	if cdnaPos < 0 {
 		return answer, errors.New("input cDNA position must be positive")
 	}
-	if cdnaPos > len(g.cdnaSeq)-1 {
+	if cdnaPos > len(g.codingSeq.seq)-1 {
 		return answer, errors.New("input position is greater than the length of the cDNA")
 	}
 
 	switch cdnaPos % 3 {
 	case 0:
-		return dna.Codon{Seq: g.cdnaSeq[cdnaPos : cdnaPos+3]}, nil
+		return dna.Codon{Seq: g.codingSeq.seq[cdnaPos : cdnaPos+3]}, nil
```

Comment:
This relates to last two comments.  cdna seems to make sure sense than codingSeq.  Maybe I am not understanding what the codingSeq is?

Reply:
Before I was using cDNA to refer to the actual coding sequence of the gene, but of course the cDNA also includes the UTR sequences, so I made several changes to make cdna and codingSeq different.